src/aspire/nufft/__init__.py

```diff
@@ -141,7 +141,7 @@ def __new__(cls, *args, **kwargs):
             return super(Plan, cls).__new__(cls)
 
 
-def anufft(sig_f, fourier_pts, sz, real=False):
+def anufft(sig_f, fourier_pts, sz, real=False, epsilon=1e-8):
```

Comment:
Just curious as to why this was added. I believe there was discussion but I can't recall.

Reply:
The precision guarantees based on `epsilon` (as a parameter of `FLEBasis2D`) depend on that same `epsilon` being the precision of the NUFFT. Prior to adding this option for `nufft` and `anufft`, there was no way to ask for a specific tolerance from them. (in particular, it was impossible to get past whatever tolerance was hardcoded in)

Reply:
Ok, thanks for the explanation!